server/src/db.py

```diff
@@ -22,6 +24,7 @@ def db() -> motor_asyncio.AsyncIOMotorDatabase:
 
 
 async def create_room(data: dict):
+    data['password'] = encrypt(key=SECRET, msg=data['password'])
```

Comment:
IMHO is better to import settings in the encrypt.py, not pass it like function argument, also check how it will work, if password will be empty